ush/soca/prep_marine_obs.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+
+from wxflow import FileHandler
+import os
+import fnmatch
+
+
+DMPDIR = os.getenv('DMPDIR')
+cyc = os.getenv('cyc')
+PDY = os.getenv('PDY')
+CDUMP = os.getenv('CDUMP')
+COMIN_OBS = os.getenv('COMIN_OBS')
+
+cycdir = os.path.join(DMPDIR, CDUMP + '.' + str(PDY), str(cyc))
+
+# TODO: this looks good for a yaml
+obs_dict = {
+    #                   'rads_adt_3a_2021182.nc'
```

Comment:
Did you forget to remove the comments? If so, remove, here and below.

Reply:
I think it would help for future troubleshooting